### DISCUSSION TITLE
Should I mark Iden as Copy?

### DISCUSSION THREAD

**Discussion by ghost** [SeaQL/sea-query#649] [General]
Hey peeps,

Due to the use of static like this

```
pub static MEMBERSHIP_TABLE: (Schemas, Memberships) = (Schemas::Public, Memberships::Table);
```

I would like to mark `Schemas` and `Memberships` as `Copy` so that I don't have to `MEMBERSHIP_TABLE.clone()` when I create a statement.

I wonder whether it is the correct way or there are other ways?

Cheers,

**Comment by billy1624** [comment]
Hey @phungtranminh, I think you can. And it's perfectly okay to do so.

**Comment by ghost** [comment]
Thanks! @billy1624
